content/_includes/post.macros.njk

```diff
@@ -245,15 +247,24 @@ params:
     default: true
     note: Optionally remove event links for content in linked-cards
 #}
-{% macro upcoming(
+{% macro eventpreview(
   events,
   links=true
 ) %}
-  {%- set events = events | getFuture if events else none -%}
+  {%- set events = events -%}
   {%- set events = events | sort(attribute='date') -%}
-  {%- if events | length > 0 -%}
+  {%- set pastevents = events -%}
+  {%- set futureevents = events | getFuture  -%}
+  {%- if futureevents | length > 0 -%}
+    <h4>Upcoming</h4>
+  {%- elif pastevents | length > 0 -%}
+    <h4>Past</h4>
+  {%- endif -%}
+
+  {%- if events and (events | length > 0) -%}
     <ul class="next-event">
-      {%- for next in events -%}
+    {%- for next in events -%}
```

Comment:
When this is a list of past events, I think we want to reverse the order so that it shows the two _most recent_ dates instead of the two _oldest_ dates.

![Screen Shot 2021-04-15 at 9 17 56 AM](https://user-images.githubusercontent.com/552316/114875658-81690f00-9dcb-11eb-98c1-aa8804a3885b.png)

Reply:
Commit [8a6e35c](https://github.com/oddbird/oddleventy/pull/139/commits/8a6e35c0873c5ab866b22bea59ca8bbbaecab0b7) adjusts the order of past events so that the most recent one appears first. @jgerigmeyer @stacyk 

It works best for talks/events where there were at least two in the same year. 

For past events where the two most recent ones took place in separate years, it makes it more clear that if we go this route we will have to adjust how we display years. 

Here, for the Agile Design Systems talk, the first event was in 2019 but the second one is from 2018
![image](https://user-images.githubusercontent.com/41588129/115023535-39c3af80-9ebf-11eb-87d0-8aa6d0e56e2e.png)